modules/bitgo/test/v2/unit/coins/utxo/transaction.ts

```diff
@@ -44,10 +43,30 @@ function run<TNumber extends number | bigint = number>(
   amountType: 'number' | 'bigint' = 'number'
 ) {
   describe(`Transaction Stages ${coin.getChain()} (${amountType}) scripts=${inputScripts.join(',')}`, function () {
-    const wallet = getUtxoWallet(coin);
-    const walletKeys = getDefaultWalletKeys();
+    const isTransactionWithKeyPathSpend = inputScripts.some((s) => s === 'p2trMusig2');
+    const isPsbt = isTransactionWithKeyPathSpend;
+    const isTransactionWithReplayProtection = inputScripts.some((s) => s === 'replayProtection');
```

Comment:
Let's make `isPsbt` a new parameter here instead. We want to cover the other input scripts as well.

Reply:
Done